NEWS.md

```diff
@@ -16,4 +16,67 @@
+  `wallinga_teunis`, and `DiscrSI` becomes `discr_si`. Names of arguments to
+  these functions have also changed to snake_case. Note that compatibility
+  functions have been added so that the old functions as written in EpiEstim
+  1.1-0 should still work but throw a warning pointing to the newest functions. 
+* Compatibility with `incidence` package: in the function `estimate_R`, the
+  first argument, i.e. the incidence from which the reproduction number is
+  calculated, can now be, either a vector of case counts (as in version 1.1-0) or
+  an `incidence` object (see R package `incidence`).
+* Accounting for imported cases: in the function `estimate_R`, the first
+  argument, i.e. the incidence from which the reproduction number can now
+  provide information about known imported cases: by specifying the first
+  argument as either a dataframe with columns "local" and "imported", or an
+  `incidence` object with two groups (local and imported, see R package
+  `incidence`). This new feature is described in Thompson et al. Epidemics 2019
+  (currently in review).
+* Additional methods available for function `estimate_R`: in addition to
+  `non_parametric_si`, `parametric_si` and `uncertain_si`, which were already
+  available in EpiEstim 1.1-0, two new methods have been added: `si_from_data` or
+  `si_from_sample`. These allow feeding function `estimate_R` data on observed
+  serial intervals (method `si_from_data`) or posterior samples of serial
+  interval distributions obtained from such data (method `si_from_sample`). These
+  new features are described in Thompson et al. Epidemics 2019 (currently in
+  review).
+* No more plotting option inside of `estimate_R`: estimate_R now generates on
+  object of class `estimate_R`, which can be plotted separately by using the
+  new `estimate_R_plots` function, which also now allows to plot several R
+  estimates on a single plot. 
+* New argument `config` for `estimate_R` function: this is meant to minimise
+  the number of arguments to function `estimate_R`; so arguments `method`,
+  `t_start`, `t_end`, `n1`, `n2`, `mean_si`, `std_si`, `std_mean_si`,
+  `min_mean_si`, `max_mean_si`, `std_std_si`, `min_std_si`, `max_std_si`,
+  `si_distr`, `mean_prior`, `std_prior`, and `cv_posterior` are now specified as
+  a group under this new `config` argument. Such a `config` argument must be of
+  class `estimate_R_config` and can be obtained as a results of the new
+  `make_config` function. 
+* New function `make_config`, which defines settings for function `estimate_R`,
+  and sets defaults where arguments are missing. In particular, if argument
+  `incid` is not `NULL`, by default `config$t_start` and `config$t_end` will be
+  set so that, when the configuration is used inside `estimate_R` function, the
+  reproduction number is estimated by default on sliding weekly windows (in
+  EpiEstim 1.1-0 there was no default for the time window of estimation of R).
+
```

Comment:
You might want to add a section about new vignette documentation.

Reply:
thanks for the suggestion - done